### ISSUE TITLE
New Vulcan version didn't work on Ubuntu linux

### ISSUE THREAD

**Issue by Trapezozo** [ValveSoftware/Source-1-Games#5757]
I can play only on OpenGL version, but Vulcan crash on TF2 loading screen. OpenGL TF2 is full of stutters.

**Comment by TheEpicNoobZilla** [comment]
I am not valve employee, but could you share your PC specs?
It should be under "About" page on Settings

**Comment by Trapezozo** [comment]
> I am not valve employee, but could you share your PC specs? It should be under "About" page on Settings

ThinkPad L490 (IGPU)

**Comment by NULLYUKI** [comment]
Related to https://github.com/ValveSoftware/Source-1-Games/issues/5630

**Comment by AHuski** [comment]
I'm having the same problem on Debian. When I launch the vulkan version, I just see a black screen. However, if I turn off my monitor and turn it back on, the game works, but it changes my display refresh rate to 24Hz. Then, if I queue into a game, it crashes.

**Comment by JulianHeuser** [comment]
> I'm having the same problem on Debian. When I launch the vulkan version, I just see a black screen. However, if I turn off my monitor and turn it back on, the game works, but it changes my display refresh rate to 24Hz. Then, if I queue into a game, it crashes.

Does launching with the -windowed launch option fix this for you? I had similar issues with the game trying to  "take over" the monitor settings when running in fullscreen.

**Comment by HurricanePootis** [comment]
> > I'm having the same problem on Debian. When I launch the vulkan version, I just see a black screen. However, if I turn off my monitor and turn it back on, the game works, but it changes my display refresh rate to 24Hz. Then, if I queue into a game, it crashes.
> 
> Does launching with the -windowed launch option fix this for you? I had similar issues with the game trying to "take over" the monitor settings when running in fullscreen.
> 
> Edit: If so, #5630 and #5649 seem to indicate that there's something really weird with tf2's fullscreen on Linux after the 64 bit update. These all seem to be related in some way.

Actually, for me, it turned out to be a kwin issue. I am not sure if tf2 changed something to cause this problem, or kwin changed something to cause this problem.

**Comment by AHuski** [comment]
> Does launching with the -windowed launch option fix this for you? I had similar issues with the game trying to "take over" the monitor settings when running in fullscreen.
>
> Edit: If so, https://github.com/ValveSoftware/Source-1-Games/issues/5630 and https://github.com/ValveSoftware/Source-1-Games/issues/5649 seem to indicate that there's something really weird with tf2's fullscreen on Linux after the 64 bit update. These all seem to be related in some way.

Yes, using -windowed works. My issue seems to be exactly the same as #5649.

**Comment by Trapezozo** [comment]
Using -windowed allowed me to launch this game. But on Valve casual I got terrible performance. So I better stay on OpenGL. Classic Valve's update.

**Comment by toast-ts** [comment]
Yes, I can also confirm that using `-windowed` allowed me to actually open the game and switching to fullscreen using the settings. It seems to crash the game (and my GNOME session somehow..) if I don't pass `-windowed` to the launch parameters, and I am also on X11 as Wayland doesn't work on my system.
